### PR TITLE
Ci/feature/mrti 6404

### PR DESCRIPTION
**Pull request by cert-orangecyberdefense**

### CODE REVIEW

File: datalake/endpoints/comments.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        }
+        url = self._build_url_for_endpoint("comments").format(hashkey=hashkey)
+        response = self.datalake_requests(url, "post", self._post_headers(), payload)
+        return parse_response(response)
+
+    def post_comments(
+        self, hashkeys: List[str], comment: str, public=True
+    ) -> Tuple[list, list]:
+        """
+        Post same comment on on one or more threats (hashkeys)
```
Comment (BFachon) [inline datalake/endpoints/comments.py:27]: 2 "on"

File: datalake/endpoints/comments.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        Returns two lists: the first containing hashkeys of threats for which the comment could be added and the second when it could not
+        """
+        visibility = "public" if public else "organization"
```
Comment (BFachon) [inline datalake/endpoints/comments.py:30]: description_visibility?

File: datalake_scripts/scripts/get_threats.py [hunk 3]
```diff
@@ -23,36 +21,35 @@ def main(override_args=None):
...
     parser.add_argument(
         "-i",
-        "--input_file",
+        "--input",
         help="list of threats ids (hashkeys) that need to be retrieved",
     )
     parser.add_argument(
         "--lost",
-        help="saved hashes that were not found",
+        help="file path to save hashkeys that were not found into",
```
Comment (BFachon) [inline datalake_scripts/scripts/get_threats.py:29]: I didn't fully understand

**Review by BFachon — COMMENTED** [review]
[tests_feedback.txt](https://github.com/user-attachments/files/17541327/tests_feedback.txt)

**Comment by ClaireOrange** [comment]
my notes on your feedbacks 
[tests_feedback.txt](https://github.com/user-attachments/files/17622574/tests_feedback.txt)

**Comment by BFachon** [comment]
Here is a second feedback : 
[second_tests_feedback.txt](https://github.com/user-attachments/files/17662615/second_tests_feedback.txt)

**Comment by ClaireOrange** [comment]
issue on tag not linked to client

**Review by BFachon — APPROVED** [review]

**Pull request merged**